backend/tracim_backend/app_models/email_validators.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+from marshmallow import ValidationError
+from marshmallow.validate import Email
+
+
+class TracimEmailValidator(Email):
+    """ Special Email Validator who accept email from subdomain"""
```

Comment:
...and many other things really

```suggestion
    """ Email Validator accepting a wider range of email addresses than marshmallow's Email field"""
```